docs/docs/configuration/index.md

```diff
@@ -477,6 +477,18 @@ cameras:
       order: 0
       # Optional: Whether or not to show the camera in the Frigate UI (default: shown below)
       dashboard: True
+      # Optional: Override browser locale and show time in 12 / 24 hour format (default: use browser locale)
+      use12hour: False
+      # Optional: Set the date style for a specified length. 
+      # Options are: full, long, medium, sort (default: shown below).
+      date_style: short
```

Comment:
Nobody is going to know what full, long, medium, short mean. We should include an example of each.